tests/plugin_e2e/test_minimax_plugin.py

```
@@ -0,0 +1,155 @@
+
+    def rpc(payload: dict) -> dict:
+        request = urllib.request.Request(
+            "http://127.0.0.1:14242/mcp/",
+            data=json.dumps(payload).encode("utf-8"),
+            headers={
+                "Content-Type": "application/json",
+                "Accept": "application/json, text/event-stream",
+            },
+            method="POST",
+        )
+        with urllib.request.urlopen(request, timeout=10) as response:
```

Ruff (0.16.1)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)